Multiple small fixes

tobloef opened the pull request:
Be careful when merging!

tobloef committed:
Update 01_Introduction.md

tobloef committed:
Update 10_System.md

tobloef committed:
Update 11_Systems-Design.md

tobloef committed:
Update 12_Systems-Architecture.md

tobloef committed:
Update 40_Lessons.md

tobloef committed:
Update 42_Lessons-Operation.md

tobloef committed:
Update 43_Lessons-Maintaince.md

tobloef committed:
Update 15_Systems-Current-State.md

tobloef committed:
Anti-dab measures

Det er lidt at sætte os selv up for failure at skrive dette imo, da vi ikke helt ved om det er sandt. Vil ikke have Paolo dabber på os.

tobloef committed:
Update 24_Process-Repository-Organization.md

tobloef committed:
Removed "proper" reference

We don't seem to use this anywhere else, so I thought it would be best to be consistent.

Jakobis commented on report/11_Systems-Design.md:
```diff
@@ -1,19 +1,19 @@
 ### System Design 
-The MiniTwit application is a web app written in C# using ASP.NET. Data is stored using EntityFrameworkCore as a database abstraction layer working with a Postgres database. This stack was chosen since it is a well supported and maintained stack with plenty of available documentation. This means that many third-party packages are available, like SeriLog, which makes interaction with ElasticSearch ...
+The new version of the MiniTwit application is a web app written in C# using ASP.NET. Data is stored using Entity Framework Core (EF Core) as a database abstraction layer working with a PostgreSQL database. This stack was chosen because it is both well supported and maintained, as well as well documented. Furthermore, this means that many third-party packages are available, like SeriLog which mak ...
```
I do not think we should call it the new version, since a current version will always be the new version.

Jakobis replied:
Also, the sentence "well supported and maintained, as well as well documented." needs fewer wells. Perhaps just "well supported, maintained, and documented."

tobloef replied:
It was to create a distinction between the original Python version and our ASP.NET version. Do you have any ideas of how to accomplish this in a more clear way?

tobloef committed:
Update 27_Process-Monitoring.md

tobloef committed:
Update 28_Process-Logging.md

tobloef committed:
"Fixed" reference

This is the second time I've seen this style, so I'm starting to regret removing it. If this is some fuckery, we should revert this commit.

tobloef committed:
Update 11_Systems-Design.md

tobloef committed:
Update 11_Systems-Design.md

Hjaltesorgenfrei commented on report/12_Systems-Architecture.md:
```diff
@@ -1,17 +1,17 @@
 #### Sub systems
 
-Traefik is used as a reverse proxy and Edge Router. As can be seen from the diagram it interacts with MiniTwit.Web.App, Kibana and Grafana. These three are the public facing applications and Traefik is used to manage SSL certificates and HTTPS. Load balancing can be done through Traefik, when using Docker Swarm.
+[Traefik](https://containo.us/traefik/) is used as a reverse proxy and edge router. As shown in figure X (TODO: Update) it interacts with the three public facing applications of the system, MiniTwit.Web.App, Kibana and Grafana, it provides SSL certificates management and HTTPS. Load balancing can also be done through Traefik, when using Docker Swarm.
```
; after system maybe?

Hjaltesorgenfrei commented on report/25_Process-Branching-Strategy.md:
```diff
@@ -1,6 +1,6 @@
 ## Branching Strategy
 
-Due to the smaller scope and team size, it was decided to use a lightweight, but surprisingly useful, branching strategy called GitHub flow (Github, https://guides.github.com/introduction/flow/, visited: 2020-05-10) (not to be confused with Git flow). GitHub flow works like this:
+Due to the smaller scope and team size, it was decided to use a lightweight, but surprisingly useful, branching strategy called [GitHub flow](https://guides.github.com/introduction/flow/) (not to be confused with Git flow). GitHub flow works like this:
```
Why is it suprisingly useful?

Hjaltesorgenfrei commented on report/27_Process-Monitoring.md:
```diff
@@ -1,16 +1,16 @@
 This was, in the early stages, all the monitoring the system had.
 
-In order to monitor more closely what each process is doing, the project utilizes [Prometheus](https://prometheus.io/docs/introduction/overview/). Prometheus is a systems monitoring toolkit, capable of gathering and storing information about processes running as well as being extended to monitor even more specific things, based on what is running on the system. Standard metrics for monitoring inc ...
+In order to monitor more closely what each process is doing, the project utilizes [Prometheus](https://prometheus.io/docs/introduction/overview/). Prometheus is a systems monitoring toolkit, capable of gathering and storing information about processes running as well as being extended to monitor even more specific things, based on what is running on the system. Standard metrics for monitoring inc ...
```
> Standard metrics for monitoring include simple low-level information, such CPU, memory and disk usage for both the system and the specific process.

What is the purpose of this line? This is not standard?

Hjaltesorgenfrei commented on report/43_Lessons-Maintaince.md:
```diff
@@ -1,9 +1,8 @@
-This issue, as well as similar ones, could probably have been caught in the pull request, if a more rigorous and defined review process was defined. 
-A more defined review process could also have alleviated the problem of not knowing what to look for when going through a pull request, which was a frequent problem over the course of this project.
\ No newline at end of file
+This issue, as well as similar ones, could probably have been caught in the pull request, if a more rigorous and well-defined review process was defined. A tighter review process could also have alleviated the problem of not knowing what to look for when going through a pull request, which was a frequent problem over the course of this project.
```
swap tighter for something else. Its weird there. 
Its the review process being more well defined that would have fixed the problem.

tobloef committed:
Update 12_Systems-Architecture.md

tobloef committed:
Update 25_Process-Branching-Strategy.md

tobloef committed:
Update 27_Process-Monitoring.md

tobloef committed:
Update 43_Lessons-Maintaince.md
